### PR TITLE
feat: create Tooltip variant for Popover

### LINKED ISSUE #18: Create Tooltip variant for Popover

### PR DESCRIPTION
**Pull request by massimo-caser**
resolve #18

### CODE REVIEW

File: src/components/Tooltip/Tooltip.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.tooltip {
+    @apply absolute z-10
+}
+
+.tooltip__container {
+    @apply relative
```
Comment (karim-hajouan) [inline src/components/Tooltip/Tooltip.css:6]: is this `relative` really needed ?

File: src/components/Popover/Popover.tsx [hunk 2]
```diff
@@ -34,17 +36,44 @@ function Popover({
       <HeadlessPopover.Button
         className="focus-visible:outline-0"
         ref={setReferenceElement}
+        onMouseEnter={() => {
+          if (tooltip) {
+            console.log(tooltip);
```
Comment (ilyichv) [inline src/components/Popover/Popover.tsx:41]: remove

File: src/components/Popover/Popover.tsx [hunk 3]
```diff
@@ -34,17 +36,44 @@ function Popover({
...
+      ) : (
+        <HeadlessPopover.Panel
+          {...attributes.popper}
+          style={styles.popper}
+          ref={setPopperElement}
+          className={twMerge("tooltip", className)}
+        >
+          {popover}
+        </HeadlessPopover.Panel>
+      )}
```
Comment (ilyichv) [inline src/components/Popover/Popover.tsx:76]: Instead of using it that way, can't we just handle both cases with the `show` variable?

**Review by ilyichv — APPROVED** [review]

**Pull request merged**